druid.md

```diff
@@ -24,19 +24,17 @@ provides a framework for plugging in new functionality. Druid also supports
 approximate algorithms for cardinality estimation, and histogram and quantile
 calculations.
 
-**Interactive Queries** Druid’s query latency is optimized by reading and
-scanning only exactly what is needed. Aggregate and filter on data without
-sitting around waiting for results. Druid is ideal for powering analytic
-dashboards. Druid is a column-oriented data store and supports ad-hoc,
-  multi-dimensional filtering.
+**Sub-second Queries** Druid’s column orientation and inverted indexes enable
+scanning exactly what is needed for a query. Aggregate and filter on data in
+milliseconds. Druid is ideal for powering analytic dashboards. 
```

Comment:
Why remove ad-hoc multi-dimensional filtering? Many other things out there don't do it well.